docs/Releasing-the-plugin.md

```diff
@@ -24,24 +39,49 @@ In addition to those locations, run the `npm run since -- -r {version}` command
 
 The module headers from the plugin have to be translated using a separate `module-i18n.php` file which can be automatically generated and updated. Run `npm run translations` to update the file to reflect the latest available modules.
 
-### Update readme.txt
+### Update `readme.txt`
 
-Run `npm run readme -- -m "{milestoneName}"` to update the readme.txt file with the release changelog, where `{milestoneName}` is the name of the milestone. For example, if the milestone is `1.2.0`, the command needs to be `npm run readme -- -m "1.2.0"`.
+Run `npm run readme -- -m "{milestoneName}"` to update the `readme.txt` file with the release changelog, where `{milestoneName}` is the name of the milestone. For example, if the milestone is `1.2.0`, the command needs to be `npm run readme -- -m "1.2.0"`.
 
-After running the command, check the readme.txt file to ensure the new changelog for the release has been added. Also review its changelog entries for whether they make sense and are understandable.
+After running the command, check the `readme.txt` file to ensure the new changelog for the release has been added. Also review its changelog entries for whether they make sense and are understandable. Make sure that no unexpected changelog entries are present.
 
 ### Open a pull request
 
-Push your local release branch to the GitHub repository and open a pull request against `trunk`. Make sure to tag at least 2 plugin maintainers to request a review. Once the pull request is approved by at least 2 plugin maintainers, the pull request can be merged.
-
-If this is a major or minor release, please keep the release branch around since that branch can be used to later create patch releases from the same state of the codebase.
+Push your local branch to the GitHub repository and open a pull request against the release branch `release/{milestoneName}`. Make sure to tag at least 2 plugin maintainers to request a review, as usual. Once the pull request is approved by at least 2 plugin maintainers, the pull request can be merged.
```

Comment:
"As usual, make sure to request a review from 2 plugin maintainers. Once approved, the pull request can be merged."